src/sql/workbench/services/connection/test/browser/connectionManagementService.test.ts

```diff
@@ -854,6 +870,21 @@ suite('SQL ConnectionManagementService tests', () => {
 				properties: undefined
 			}
 		]));
+
+		accountManagementService.setup(x => x.getAccounts()).returns(() => {
+			return Promise.resolve<azdata.Account[]>([
+				{
+					key: {
+						accountId: username,
+						providerId,
+					},
+					displayInfo: undefined,
+					isStale: false,
+					properties: undefined
+				}
+			]);
+		});
+
 		let testToken = 'testToken';
 		let returnedTokens = {};
 		returnedTokens['azurePublicCloud'] = { token: 'badToken' };
```

Comment:
should this be `azure_PublicCloud` ?

Reply:
Realized it's a test, but still good to be consistent

Reply:
Definitely!